### DISCUSSION TITLE
Ability to change default annotations

### DISCUSSION THREAD

**Discussion by rmgpinto** [grafana/grafana#40513] [Dashboards, Panels & Visualizations]
AFAIK, grafana adds the following default annotation to each dashboard:

```json
annotations: {
  list: [
    {
        "builtIn": 1,
        "datasource": "-- Grafana --",
        "enable": true,
        "hide": true,
        "iconColor": "rgba(0, 211, 255, 1)",
        "name": "Annotations & Alerts",
        "type": "dashboard"
    }
  ]
}
```

Would it be possible to change the default annotations? I wanted to replace `type: dashboard` with `type: tags` and have all annotations appear on all dashboards by default.
Thanks

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
